tsconfig.json

```diff
@@ -9,8 +9,9 @@
         "newLine": "LF",
         "noUnusedLocals": true,
         "noUnusedParameters": true,
-        "strict": true
+        "strict": true,
+        "skipLibCheck": true
```

Comment:
do we realy need it?

Reply:
This is to bypass the libcheck for vite, it throws a lot of error. Since we dont use vite for compilation, but only for testing

Reply:
OK makes sense.